python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -211,6 +211,38 @@ def start_sync(self, connector_id: str) -> Dict[str, Any]:
         )
         return connector_details
 
+    def start_resync(
+        self, connector_id: str, resync_parameters: Dict[str, List[str]]
+    ) -> Dict[str, Any]:
+        """
+        Initiates a historical sync of all data for multiple schema tables within a Fivetran connector.
+
+        Args:
+            connector_id (str): The Fivetran Connector ID. You can retrieve this value from the
+                "Setup" tab of a given connector in the Fivetran UI.
+            resync_parameters (Dict[str, List[str]]): The resync parameters to send to the Fivetran API.
+                An example payload can be found here: https://fivetran.com/docs/rest-api/connectors#request_6
+
+        Returns:
+            Dict[str, Any]: Parsed json data representing the connector details API response after
+                the resync is started.
+        """
+        if self._disable_schedule_on_trigger:
+            self._log.info("Disabling Fivetran sync schedule.")
+            self.update_schedule_type(connector_id, "manual")
+        self._assert_syncable_connector(connector_id)
+        self.make_request(
+            method="POST",
+            endpoint=f"{connector_id}/schemas/tables/resync",
+            data=json.dumps(resync_parameters),
+        )
+        connector_details = self.get_connector_details(connector_id)
+        self._log.info(
+            f"Sync initialized for connector_id={connector_id}. View this sync in the Fivetran UI: "
```

Comment:
nit: this should say resync instead of sync

Reply:
@OwenKephart resolved 👍